modules/oe_content_news/oe_content_news.post_update.php

```diff
@@ -57,3 +57,28 @@ function oe_content_news_post_update_00004(): void {
     $updated_form_display->save();
   }
 }
+
+/**
+ * Create oe_news_contacts field in the news content type.
+ */
+function oe_content_news_post_update_00005(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_news') . '/config/post_updates/00005_create_fields');
+  \Drupal::service('config.installer')->installOptionalConfig($storage);
+}
+
+/**
+ * Update news node form display.
+ */
+function oe_content_news_post_update_00006(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_news') . '/config/post_updates/00006_update_form_display');
+
+  // Form display configurations to update.
+  $form_display_values = $storage->read('core.entity_form_display.node.oe_news.default');
```

Comment:
Please update in core.entity_form_display.node.oe_news.default 00004_update_form_display and we can get rid of oe_content_news_post_update_00006()

Reply:
updated